draft-bctb-6man-rfc6296-bis.md

```diff
@@ -90,7 +90,7 @@ ramifications:
 * Any security benefit that NAPT44 might offer is not present in
       NPTv6, necessitating the use of a firewall to obtain those
       benefits if desired.  An example of such a firewall is described
-      in {{?RFC6092}}.
+      in {{?RFC692}}.
```

Comment:
You probably don't want to make these leading zero changes to the RFC references. :-)

Reply:
ha ha ha. 

I don't know how this was changed, but I updated the PR.